src/lms-harmonizer/edfi_lms_harmonizer/scripts/mssql/0011_harmonize_assignment_submissions.sql

```diff
@@ -84,12 +88,21 @@ BEGIN
 				submsisionstatusdescriptor.Namespace = 'uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/Schoology'
 			AND
 				submsisionstatusdescriptor.CodeValue = 'missing'
-		) as submsisionstatusdescriptor
+		) as latesubmissionstatusdescriptor
+		CROSS APPLY (
+			SELECT
+				submsisionstatusdescriptor.DescriptorId
+			FROM
+				edfi.Descriptor submsisionstatusdescriptor
+			WHERE
+				submsisionstatusdescriptor.Namespace = 'uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/Schoology'
+			AND
+				submsisionstatusdescriptor.CodeValue = 'Upcoming'
+		) as upcomingsubmissionstatusdescriptor
 		WHERE NOT EXISTS (
 			SELECT 1 FROM lms.AssignmentSubmission lmssubmission WHERE lmssubmission.AssignmentIdentifier = lmsassignment.AssignmentIdentifier
 				AND lmssubmission.LMSUserIdentifier = lmsstudent.LMSUserIdentifier
 		)
-		AND lmsxassignment.DueDateTime < GETDATE()
 		AND (edfisectionassociation.EndDate IS NULL OR enddate > lmsassignment.DueDateTime)
```

Comment:
I did not notice this before, would be better to change:
```suggestion
		AND (edfisectionassociation.EndDate IS NULL OR edfisectionassociation.EndDate > lmsassignment.DueDateTime)
```

Reply:
I tested with this change, therefore committing it now.